resolvers/relay.ts

```diff
@@ -0,0 +1,88 @@
+export type RelayInformation = {
+    name?: string;
+    description?: string;
+    pubkey?: string;
+    contact?: string;
+    supported_nips?: number[];
+    software?: string;
+    version?: string;
+    icon?: string;
+};
+
+export const Information = (kv: Deno.Kv) =>
+    async function () {
+        const res = await kv.get<RelayInformation>(["relay_information"]);
+        if (res.value == null) return null;
+        return res.value;
+    };
+
+export class RelayInformationStore {
+    information: RelayInformation = {
+        software: "https://github.com/BlowaterNostr/relayed",
+        supported_nips: [1, 2],
+        version: "RC5",
+    };
+
+    constructor(
+        default_information: RelayInformation,
+        private kv: Deno.Kv,
+        initial_information: RelayInformation | null,
```

Comment:
In the preview, default_information is the information filled in by the user when running relay, and initial_information is obtained by reading kv. But now I have changed it to only read when needed, instead of default reading.